src/workerd/api/http.h

```diff
@@ -639,14 +661,47 @@ class Request: public Body {
   jsg::Optional<v8::Local<v8::Object>> getCf(jsg::Lock& js);
   // Returns the `cf` field containing Cloudflare feature flags.
 
-  jsg::WontImplement getContext() { return jsg::WontImplement(); }
-  // This is deprecated in the spec.
-
-  jsg::WontImplement getMode()        { return jsg::WontImplement(); }
-  jsg::WontImplement getCredentials() { return jsg::WontImplement(); }
-  jsg::Unimplemented getIntegrity()   { return jsg::Unimplemented(); }
-  jsg::Unimplemented getCache()       { return jsg::Unimplemented(); }
-  // See members of Initializer for commentary on unimplemented APIs.
+  bool getKeepalive() { return false; }
+  // We do not implement support for the keepalive option but we do want to at least provide
```

Comment:
Why do we want to provide this, if we don't implement it?

I'm not necessarily against it, I just don't understand in what situation it makes a difference. We are non-conformant either way.

Reply:
We *are* conformant with the `keepalive = false` case. Technically we *do* implement the `false` case.